app/javascript/css/forms.scss

```diff
@@ -217,6 +217,14 @@ input[type='date']:invalid::-webkit-datetime-edit {
   overflow: hidden;
 }
 
+.income-padding {
```

Comment:
I was able to get the forms/fake forms to match up in width without adding these classes. Do you need these classes?